src/otk/context.py

```diff
@@ -8,13 +8,10 @@
 from abc import ABC, abstractmethod
 from typing import Any, Optional
 
-from .error import (
-    TransformDefineDuplicateError,
-    TransformVariableIndexRangeError,
-    TransformVariableIndexTypeError,
-    TransformVariableLookupError,
-    TransformVariableTypeError,
-)
+from .error import (TransformDefineDuplicateError,
```

Comment:
(super nitpick) I wonder if this `otk: isort context.py`  commit could be squashed into some other commit?

Reply:
I'd rather keep codestyle / formatting changes in separate commits so it doesn't look like that imports changed in a commit that has nothing to do with import changes.